app/lib/time_limit_config.rb

```diff
@@ -21,6 +21,10 @@ def to_s
 
   Rule = Struct.new(:from_date, :to_date, :limit)
 
+  def self.referee_chaser_limit
+    2
+  end
+
```

Comment:
I think I would interpret this as the maximum number of chaser emails that candidates can send, rather than the minimum amount of time between chaser emails sent?

Could we perhaps rename it, and maybe use `48.hours` instead of `2.days` just to be clear that it's a time period rather than a date period?

```suggestion
  def self.minimum_time_between_chaser_emails
    48.hours
  end

```

Reply:
We can make the name change. @stevehook and I couldn't decide what to call it 😂 

I'm not so sure about changing it to hours though because all of the others in `app/lib/time_limit_config.rb` are in days. I think it would make it confusing. 

If you want, we could extract it out of the file and call the time directly instead of making a method. I'm not too fussed either way. What do you think @frankieroberto?

Reply:
or maybe we could rename the method so that its `minimum_hours_between_chaser_emails`?

Reply:
Discussed offline - changed to hours and renamed method.